tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -12760,12 +12755,26 @@ node_modules/chrome-devtools-frontend/front_end/ui/XElement.js(108,1): error TS8
 node_modules/chrome-devtools-frontend/front_end/ui/XElement.js(121,1): error TS8022: JSDoc '@extends' is not attached to a class.
 node_modules/chrome-devtools-frontend/front_end/ui/XElement.js(131,1): error TS8022: JSDoc '@extends' is not attached to a class.
 node_modules/chrome-devtools-frontend/front_end/ui/XElement.js(141,1): error TS8022: JSDoc '@extends' is not attached to a class.
+node_modules/chrome-devtools-frontend/front_end/ui/XElement.js(149,38): error TS2345: Argument of type 'typeof XVBox' is not assignable to parameter of type 'CustomElementConstructor'.
```

Comment:
@sandersn it all looks like renames/moves except for this - any idea which of our changes caused it?

Reply:
It is a stricter type than before, but it is  incorrect — I fixed it yesterday.